### ISSUE TITLE
build broken with docker/pkg/archive HEAD

### ISSUE THREAD

**Issue by proppy** [fsouza/go-dockerclient#197]
```
proppy@noir ~/s/g/f/go-dockerclient> go test .
# github.com/fsouza/go-dockerclient
./tar.go:30: unknown archive.TarOptions field 'Excludes' in struct literal
FAIL    github.com/fsouza/go-dockerclient [build failed]
```

Likely because of https://github.com/docker/docker/commit/6d801a3caa54ad7ef574bc426aa1ffc412c5af82

**Comment by proppy** [comment]
Fixed by either #194 #195 or #196

**Comment by fsouza** [comment]
Hey @proppy, sorry for the delay, I'm on vacations right now, so I was afk these days, but now I'm kinda back! :) 

Thanks for contributing, and for the Twitter alert.
